pkg/cfghandler/pqsupdate.go

```diff
@@ -79,18 +80,17 @@ func PostPqsUpdate(ctx *fasthttp.RequestCtx) {
 }
 
 func SavePQSConfigToRunMod(filepath string, pqsEnabled bool) error {
+	var configData common.RunModConfig
+
 	existingConfig, err := config.ReadRunModConfig(filepath)
 	if err != nil {
-		log.Errorf("SavePQSConfigToRunMod: Unable to read existing config from %s, using defaults", filepath)
-		existingConfig.QueryTimeoutSecs = config.DEFAULT_TIMEOUT
+		log.Errorf("SavePQSConfigToRunMod: Using defaults as couldn't read config: %v", err)
+		configData = config.GetDefaultRunModConfig()
+	} else {
+		configData = existingConfig
```

Comment:
nit: this will be slightly cleaner (you don't need the `existingConfig` or the `else` block)
```
	configData, err := config.ReadRunModConfig(filepath)
	if err != nil {
		log.Errorf("SavePQSConfigToRunMod: Using defaults as couldn't read config: %v", err)
		configData = config.GetDefaultRunModConfig()
	}
```